crates/red_knot_python_semantic/src/db.rs

```diff
@@ -60,6 +62,7 @@ pub(crate) mod tests {
                 file_system: TestFileSystem::Memory(MemoryFileSystem::default()),
                 events: std::sync::Arc::default(),
                 vfs: Vfs::with_stubbed_vendored(),
+                typeshed_versions: TypeshedVersions::from_str("").unwrap(),
```

Comment:
Nit: Maybe add a `Default` or `empty` function to avoid this spreading everywhere